pandas/tests/indexes/test_base.py

```diff
@@ -1494,6 +1494,15 @@ def test_drop_tuple(self, values, to_drop):
             with pytest.raises(KeyError, match=msg):
                 removed.drop(drop_me)
 
+    def test_drop_with_duplicates_in_index(self, index):
+        # GH38051
+        if len(index) == 0:
+            return
+        expected = index.drop(index[0]).repeat(2)
```

Comment:
this works, but ideally we'd form `expected` without using `drop`.  could do

```
index = index.unique()
index = index.repeat(2)
expected = index[2:]
result = index.drop(index[0])
```

Reply:
Had a similar idea, but the fixture contains indexes with duplicates. If drop is bad, we could use unique and index[1:] atfterwards?

Reply:
can you write your idea out explicitly? im not clear on how it is different from what i wrote

Reply:
Sorry, that does not work either. Example to illustrate the problem:
```
index = Index([1, 2, 1, 2])
```
index[1:] is then Index([2, 1, 2])
Repeating and dropping one in the index leads to
```
result = Index([2,2,2,2])
```
while index[1:].repeat(2) would be
```
expected = Index([2,1,2,2,1,2])
```